### PR TITLE
added keyspace hitrate measurement

### PR DESCRIPTION
**Pull request by tylernisonoff**

### CODE REVIEW

File: plugins/redis/redis.go [hunk 1]
```diff
@@ -197,6 +208,12 @@ func gatherInfoOutput(
 
 		acc.Add(metric, fval, tags)
 	}
+	var keyspace_hitrate float64 = 0.0
+	if keyspace_hits != 0 || keyspace_misses != 0 {
+		keyspace_hitrate = float64(keyspace_hits) / float64(keyspace_hits+keyspace_misses)
+	}
+	fmt.Println(keyspace_hitrate)
```
Comment (sparrc) [inline plugins/redis/redis.go]: This shouldn't be printing

File: plugins/redis/redis_test.go [hunk 2]
```diff
@@ -115,7 +116,7 @@ client_longest_output_list:0
 client_biggest_input_buf:0
 blocked_clients:0
 
-# Memory
+# Memor2
```
Comment (sparrc) [inline plugins/redis/redis_test.go]: sp

**Comment by sparrc** [comment]
thanks @tylernisonoff! :+1:, just a few comments

**Comment by tylernisonoff** [comment]
whoops sorry, didn't mean for those to sneak into the PR ;) I'll fix now

**Comment by tylernisonoff** [comment]
Looks good now @sparrc

**Pull request closed without merging**